packages/dd-trace/src/profiling/profilers/events.js

```diff
@@ -171,10 +171,11 @@ class EventsProfiler {
   stop () {
     if (this._observer) {
       this._observer.disconnect()
+      this._observer = undefined
```

Comment:
With this, we could also move the `.observe` call into the `if` statement in the `start ()` method.